File: 04_Exercises/Python_Solutions/simpleCalculator.py
print("=== KALKULATOR ===")
print("1. +")
print("2. -")
print("3. *")
print("4. /")

operationType = str(input("Wybierz typ operacji: "))
print("Podaj dane wejsciowe")
firstNumber = float(input("a = "))
secondNumber = float(input("b = "))

if (operationType == "1" or operationType == "+"):
    print("Suma a i b wynosi " + str(firstNumber+secondNumber))
elif (operationType == "2" or operationType == "-"):
    print("Roznica a i b wynosi " + str(firstNumber-secondNumber))
elif (operationType == "3" or operationType == "*"):
    print("Iloczyn a i b wynosi " + str(firstNumber*secondNumber))
elif (operationType == "4" or operationType == "/"):
    if (secondNumber == 0):
        print("[!] Nie mozna dzielic przez zero")
        exit()
    print("Iloraz a i b wynosi " + str(firstNumber/secondNumber))

else:
    print("Nieznany typ operacji")

File: 04_Exercises/CSharp_Solutions/simpleCalculator.cs
using System;

public class HelloWorld{
    public static void Main(string[] args){
        Console.WriteLine("=== KALKULATOR ===");
        Console.WriteLine("1. +");
        Console.WriteLine("2. -");
        Console.WriteLine("3. *");
        Console.WriteLine("4. /");
        Console.Write("Wybierz typ operacji: ");
        string operationType = Console.ReadLine();

        Console.WriteLine("Podaj dane wejsciowe");
        Console.Write("a = ");
        double firstNumber = Convert.ToDouble(Console.ReadLine());

        Console.Write("b = ");
        double secondNumber = Convert.ToDouble(Console.ReadLine());

        if (operationType == "+" || operationType == "1"){
            Console.WriteLine("Suma liczb a i b wynosi " + (firstNumber+secondNumber));
        }

        else if (operationType == "-" || operationType == "2"){
            Console.WriteLine("Roznica liczb a i b wynosi " + (firstNumber-secondNumber));
        }

        else if (operationType == "*" || operationType == "3"){
            Console.WriteLine("Iloczyn liczb a i b wynosi " + (firstNumber*secondNumber));
        }

        else if (operationType == "/" || operationType == "4"){
            if (secondNumber == 0){
                Console.WriteLine("[!] Nie mozna dzielic przez zero.");
                return;
            }
            Console.WriteLine("Iloraz liczb a i b wynosi " + (firstNumber/secondNumber));
        }
        
        else{
            Console.WriteLine("[!] Nieznany typ dzialania.");
        }
    }
}